screens/EditBaniOrder.js

```diff
@@ -10,49 +12,109 @@ import {
   Platform,
   StatusBar,
 } from "react-native";
+import PropTypes from "prop-types";
 import { Header } from "react-native-elements";
 import Icon from "react-native-vector-icons/MaterialIcons";
-import GLOBAL from "../utils/globals";
 import SortableList from "react-native-sortable-list";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
+import GLOBAL from "../utils/globals";
 import * as actions from "../actions/actions";
 import AnalyticsManager from "../utils/analytics";
-import { baseFontSize } from "../utils/helpers";
-import { defaultBaniOrderArray } from "../utils/helpers";
+import { baseFontSize, defaultBaniOrderArray } from "../utils/helpers";
 import Strings from "../utils/localization";
 
 const window = Dimensions.get("window");
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "#eee",
+
+    ...Platform.select({
+      ios: {
+        paddingTop: 0,
+      },
+    }),
+  },
+
+  list: {
+    flex: 1,
+  },
+
+  contentContainer: {
+    width: window.width,
+  },
+
+  row: {
+    flexDirection: "row",
+    alignItems: "center",
+    backgroundColor: "#fff",
```

Comment:
Can this color string be changed as well?

Reply:
fixed